splitwise/main_start.py

```diff
@@ -0,0 +1,30 @@
+from splitwise import Splitwise
+from user import User
+from constants import *
+
+
+u1 = User("gopal", "gopal@mail.com", "9687121229")
+u2 = User("yadav", "yadav@mail.com", "5656556661")
+u3 = User("karan", "karan@mail.com", "5455121229")
+users_list = [ u1, u2, u3]
+
+splitwise = Splitwise(users_list)
+splitwise.add_users(users_list)
```

Comment:
what is this for?

Reply:
This is for adding users initially, as it was mentioned in the problem statement that we can create few users .

Reply:
you are doing the same thing through the constructor as well